index.bs

```diff
@@ -4584,7 +4584,9 @@ strong [=/map=] from handle ids to their corresponding objects.
 
 <pre class="cddl remote-cddl local-cddl">
```

Comment:
I don't think this should be `local-cddl`, right? These values are never parsed by the client.

Reply:
They aren't parsed, but they are passed by the client.